lib/fog/aws/compute.rb

```diff
@@ -560,7 +560,16 @@ def _request(body, headers, idempotent, parser, retries = 0)
                   Fog::Compute::AWS::NotFound.slurp(error, match[:message])
                 when 'RequestLimitExceeded'
                   if retries < max_retries
-                    sleep (2.0 ** (1.0 + retries) * 100) / 1000.0
+                    jitter = rand(100)
+                    waiting = true
+                    start_time = Time.now
+                    wait_time = ((2.0 ** (1.0 + retries) * 100) + jitter) / 1000.0
+                    puts "Waiting #{wait_time} seconds to retry."
```

Comment:
We probably won't want a puts here. I think we should either omit or use `Fog::Logger` so that people can change levels/etc if desired, probably at debug level? What do you think?

Reply:
Ah yes, sorry about that. I agree completely.